src/ThemeHelper.php

```diff
@@ -0,0 +1,108 @@
+<?php
+
+namespace Timber;
+
+/**
+ * Class ThemeHelper
+ *
+ * Helper class to work with current & parent theme's names, paths & directories.
+ *
+ * All class methods behave just like their corresponding WordPress functions,
+ * with the noticeable difference that they won't return what comes after "/" in theme's name.
+ *
+ * For example, consider a theme with the following tree structure (2.x default child theme structure):
+ * - my-theme-name/
+ * -> images/
+ * --> foo.jpg
+ * -> theme/
+ * --> functions.php
+ * --> style.css
+ *
+ * Theme template in style.css is my-theme-name/theme
```

Comment:
Sorry, I don't really see why style.css would reside in a subdirectory? https://developer.wordpress.org/themes/basics/main-stylesheet-style-css/#location

Am I missing something?

Reply:
This is related to https://github.com/timber/starter-theme/issues/102 and the changes introduced in https://github.com/timber/starter-theme/pull/103. Apparently it’s possible to add theme files into a subdirectory of the theme.

I myself don’t use the starter theme, so I don’t have a strong opinion on this one.

Reply:
I have very mixed feelings about his. Not sure this kind of structure:
- should be handled in core
- further more, should be proposed as default in the starter theme